rewrite-maven/src/test/java/org/openrewrite/maven/UpgradePluginVersionTest.java

```diff
@@ -736,4 +736,203 @@ void defaultPluginGroupId() {
+                                     <artifactId>lombok</artifactId>
+                                     <optional>true</optional>
+                                 </dependency>
+                                 <dependency>
+                                     <groupId>org.springframework.boot</groupId>
+                                     <artifactId>spring-boot-starter-test</artifactId>
+                                     <scope>test</scope>
+                                 </dependency>
+                             </dependencies>
+             
+                             <build>
+                                 <plugins>
+                                     <plugin>
+                                         <groupId>org.springframework.boot</groupId>
+                                         <artifactId>spring-boot-maven-plugin</artifactId>
+                                         <configuration>
+                                             <excludes>
+                                                 <exclude>
+                                                     <groupId>org.projectlombok</groupId>
+                                                     <artifactId>lombok</artifactId>
+                                                 </exclude>
+                                             </excludes>
+                                         </configuration>
+                                     </plugin>
+                                     <plugin>
+                                         <groupId>org.jacoco</groupId>
+                                         <artifactId>jacoco-maven-plugin</artifactId>
+                                         <version>${jacoco-maven-plugin.version}</version>
+                                     </plugin>
+                                 </plugins>
+                             </build>
+              
+                             <profiles>
+                                 <profile>
+                                     <id>p1</id>
+                                     <build>
+                                         <plugins>
+                                             <plugin>
+                                                 <groupId>org.jacoco</groupId>
+                                                 <artifactId>jacoco-maven-plugin</artifactId>
+                                                 <version>${jacoco-maven-plugin.version}</version>
+                                             </plugin>
+                                         </plugins>
+                                     </build>
+                                 </profile>
+                             </profiles>
+             
+                         </project>
+              
+              """,
+            """
+              <?xml version="1.0" encoding="UTF-8"?>
+                         <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Let's left-trim these text blocks and then have the IntelliJ auto format pull them to the left.
```suggestion
                         <?xml version="1.0" encoding="UTF-8"?>
                         <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```